playground/README.md

```diff
@@ -2,7 +2,9 @@
 
 This is a simple web site built using the Monaco editor and the qsharp npm package.
 
-Build the entire repo by running `./build.py` in the root directory.
+## Building Playground Locally
```

Comment:
```suggestion
## Building the Playground Locally
```

Nitpicky, but we've been calling it "the playground" in speech so this reads more natural to me